docs/source/index.rst

```diff
@@ -6,12 +6,88 @@
 PySAGES: Python Suite for Advanced General Ensemble Simulations
 ===============================================================
 
-No compromises in usability and speed for advanced sampling methods!
+
+.. only:: html
+
+    |Github Actions CI|
+    |Github Actions CI Docker|
+    |Documentation Status|
+
+    .. |Documentation Status| image:: https://readthedocs.org/projects/pysages/badge/?version=latest
+        :target: https://pysages.readthedocs.io/en/latest/?badge=latest
+    .. |GitHub Actions CI| image:: https://github.com/SSAGESLabs/PySAGES/actions/workflows/ci.yml/badge.svg?branch=main
+        :target: https://github.com/SSAGESLabs/PySAGES/actions/workflows/ci.yml
+    .. |GitHub Actions CI Docker| image:: https://github.com/SSAGESLabs/PySAGES/actions/workflows/docker-ci.yml/badge.svg?branch=main
+        :target: https://github.com/SSAGESLabs/PySAGES/actions/workflows/docker-ci.yml
+
+
+Molecular dynamics (MD) simulations are powerful tools to investigate the static and dynamic properties of a given system.
+However, even with modern computer architecture and the fastest simulation software, computation time is limited and valuable.
+As a result, exploring a system by unbiased MD is insufficient to obtain good statistics, especially if the free-energy landscape is separated by high barriers.
+To address this well know drawback enhanced-sampling methods have been established.
+Typically, a configuration can be reduced to a collective variable (order parameter) and the simulation is biased based on these collective variables.
+The challenge for computer simulations is that i) almost every interesting system has its own collective variable description and ii) the implementation
+of collective variables and methods has run efficiently on modern computers, to allow reasonable insights into the observable of interest.
```

Comment:
```suggestion
of collective variables and methods has run efficiently on modern computers to allow reasonable insights into the observable of interest.
```

Also, I think that this should say "~~has~~ must run efficiently", but I'm not completely sure.

Reply:
indeed should be "has to run" / "must run"